radio/src/rtc.h

```diff
@@ -27,6 +27,9 @@
 #define SECS_PER_DAY    86400ul
 #define TM_YEAR_BASE    1900
 
+#define TIME_T_MIN      (-__LONG_MAX__)

```

Comment:
What is wrong with the original definition `# define TIME_T_MIN TYPE_MINIMUM (gtime_t)`?